traits.yml

```diff
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=../../python/openassetio_traitgen/schema.json
+# yamllint disable-line rule:document-start
+package: openassetio-mediacreation
+
+description:
+    Well-known Traits and Specifications for use in OpenAssetIO hosts and managers.
+
+traits:
+    timeline:
+        description: Traits related to timelines.
+        members:
+            Timeline:
+                description: This trait characterizes a collection of tracks that evaluate
+                             concurrently to form layers of references to media. Frequently used
+                             in non-linear editing environments such as Video and Audio post
+                             production tools.
+
+            Track:
+                description: This trait characterizes a lane or collection of media, arranged
+                             temporally such that only a single item in the collection is active
+                             at any given time. Frequently used in non-linear editing
+                             environments such as Video and Audio post production tools.
+
+            Clip:
+                description: This trait characterizes the use of some range of external media,
+                             commonly on a track or timeline. Frequently used in non-linear
+                             editing environments such as Video and Audio production tools.
+
+                             TODO(TC) Define any additional properties, and companion traits
+                             such as 'frameRange' and 'handles'.
+                properties:
+                    name:
+                        type: string
+                        description: The name of the clip.
+    content:
+        description: Traits related to abstract content.
+        members:
+          LocatableContent:
+            description: This trait characterizes an entity whose data is persisted externally
+                         to the API through data accessible via a valid URL.
+
+                         The `location` property holds the most applicable location of the
```

Comment:
[consider] This is clearly meant to be two paragraphs, but the rendered docstring joins it all into one.  You need two line breaks to accomplish that.

Reply:
Also see other comment re. using `>`.

Reply:
I wonder if this sort of thing could be forbidden by policy, do these deceptive line separations serve any sort of purpose?

Reply:
We always hope that the documentation of specs, traits and their properties is as verbose and informative as it can be. It's essential to people understanding what they are for and when to use them, as well as covering any intended interpretation of the data they hold. We definitely want to encourage more readable information architecture in this documentation, so being able to write paragraphs is certainly an advantage.

Reply:
Also IDE tooling may well expect PEP8 style docstrings to give pop-up docs etc.

Reply:
I wasn't making a case against paragraphs, just musing on whether single line separated "paragraphs" are ever helpful considering we could require a double line break and banish another little thing you have to remember to the depths.

Not important enough to discuss or implement though really, took your advice @feltech  [8c6c35e](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/15/commits/8c6c35e7f1ad1f660a4efa481b4a98096a7f68b4)